src/test/java/com/edu/ComputerNumberTest.java

```diff
@@ -0,0 +1,35 @@
+package com.edu;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+class ComputerNumberTest {
+
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"123", "456", "789", "678", "146", "964"})
+    void comNumberTrueTest(String testNumber) {
+        assertThatCode(() -> {
+            user.validUserNumber(testNumber);
+        }).doesNotThrowAnyException();
```

Comment:
```suggestion
        assertThatCode(() -> user.validUserNumber(testNumber)).
                doesNotThrowAnyException();
```
위와 같이 중괄호를 제거할 수도 있습니다.  (IDE의 기능을 통해서 **{** , **}** 가 비활성화 되어있는 것을 확인할 수 있습니다)